pkg/apis/servicecatalog/fake/const.go

```diff
@@ -0,0 +1,31 @@
+package fake
+
+const (
+	// Namespace is a name used for test namespaces
+	Namespace = "test-ns"
+	// NamespaceUID is a UID used for test namespaces
+	NamespaceUID = "test-ns-uid"
+
+	// BrokerURL is the URL used for test brokers
+	BrokerURL = "http://example.com"
+	// BrokerName is the name used for test brokers
+	BrokerName = "test-broker"
+
+	// ServiceClassName is the name used for test service classes
+	ServiceClassName = "test-serviceclass"
+	// ServiceClassGUID is the GUID used for test service classes
+	ServiceClassGUID = "SCGUID"
+	// PlanName is the name used for test plans
+	PlanName = "test-plan"
+	// PlanGUID is the GUID used for test plans
+	PlanGUID = "PGUID"
+	//NonBindablePlanName is the name used for test plans that should not be bindable
+	NonBindablePlanName = "test-unbindable-plan"
+	// NonBindablePlanGUID is the GUID used for test plans that should not be bindable
+	NonBindablePlanGUID = "UNBINDABLE-PLAN"
+
+	// InstanceName is a name used for test instances
+	InstanceName = "test-instance"
+	// InstanceGUID is the GUID used for test instances
+	InstanceGUID = "IGUID"
```

Comment:
Agree that the name of the package carries the meaning.  I suggest we call this package 'testfixtures', though, since the code here is not really a 'fake'.